azure-storage-js-v10-quickstart-v3/README.md

```diff
@@ -0,0 +1,62 @@
+---
+page_type: sample
+name: "Using JavaScript SDK v10 for Azure Storage"
+description: "Quickstart sample for the Azure Storage v10 SDK for JavaScript."
+languages:
+- javascript
+- nodejs
+products:
+- azure
+- azure-storage
+---
+
+# Quickstart - JavaScript SDK v10 for Azure Storage
+
+This repository implements the [quickstart](http://docs.microsoft.com/azure/storage/blobs/storage-quickstart-blobs-nodejs-v10) sample for the [Azure Storage v10 SDK for JavaScript](https://github.com/Azure/azure-storage-js).
+
+## Set up
+First, clone the repository on your machine:
```

Comment:
Instead of "First", "Next, Then, etc.  You should number them

1. Clone
2. Switch Folder
3. Install Dependencies.